src/Microsoft.DotNet.Wpf/src/System.Windows.Primitives/System.Windows.Primitives.csproj

```diff
@@ -0,0 +1,34 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  
+  <PropertyGroup>
+    <AssemblyName>System.Windows.Primitives</AssemblyName>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <CLSCompliant>false</CLSCompliant>
+    <Nullable>enable</Nullable>
+    <Platforms>AnyCPU</Platforms>
+
+    <!--
+      We align casing and naming with Win32 API. As such some types have all lower case names, which
+      in theory may conflict with new C# keywords in the future. Our types here are internal so end
+      users won't be impacted. If some name becomes difficult to adapt to with the @ symbol we can
+      cross that bridge when we hit it (if ever).
+    -->
+    <NoWarn>$(NoWarn);CS8981;CS3016</NoWarn>
+
+    <Deterministic>true</Deterministic>
+    <ProduceReferenceAssembly>true</ProduceReferenceAssembly>
+    <UsePublicApiAnalyzers>true</UsePublicApiAnalyzers>
+    <RootNamespace />
+    <DefaultItemExcludes>$(DefaultItemExcludes);ref\**</DefaultItemExcludes>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <MicrosoftPrivateWinFormsReference Include="System.Private.Windows.Core" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Windows.CsWin32" Version="$(MicrosoftWindowsCsWin32PackageVersion)" PrivateAssets="all" />
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="$(SystemRuntimeCompilerServicesUnsafePackageVersion)" />
```

Comment:
@JeremyKuhne I'm curious why this package reference was needed. This project targets net9.0/net10.0 which has Unsafe available inbox.

Reply:
@ViktorHofer Hmm. Maybe it isn't? I'll take a look.

Reply:
Submitted https://github.com/dotnet/wpf/pull/10404 to see what CI says.